Makefile

```diff
@@ -8,23 +8,31 @@ all:
 	@echo rst: uses pandoc to generate the README.rst file from README.md
 	@echo test: Runs the unit tests
 
+.PHONY: clean
 clean:
 	rm -f *.pyc
 	rm -f */*.pyc
 	rm -f -rf __pycache__
 	rm -f -rf */__pycache__
 	rm -f -rf py_cpuinfo.egg-info
 	rm -f -rf dist
+	rm -f -rf build
 	rm -f -rf py-cpuinfo-$(VERSION)
 	rm -f -rf py-cpuinfo-$(VERSION).tar.gz
 	rm -f -rf py-cpuinfo-$(VERSION).zip
+	rm -f -rf py_cpuinfo-$(VERSION)-py2.py3-none-any.whl
 
-build: clean
+_actual_build:
 	python3 setup.py sdist --formats=gztar,zip
+	python3 setup.py bdist_wheel --universal
```

Comment:
Overall, this looks alright and everything works within my limited testing.  One thing I'd like to note though is that you might want to put the `universal` flag in `setup.cfg` to avoid somebody accidentally building and uploading a wheel for just Python 2 or 3.